go.mod

```diff
@@ -31,6 +32,7 @@ require (
 	golang.org/x/sys v0.3.0
 	golang.org/x/term v0.3.0
 	gopkg.in/yaml.v2 v2.4.0
+	sigs.k8s.io/yaml v1.2.0
```

Comment:
Can you explain why a second yaml library is needed?

Reply:
needed since needed a "better" way to do yaml mangling during conversion
https://github.com/kubernetes-sigs/yaml